helm/fluent-logshipping-app/templates/configmap.yaml

```
@@ -210,6 +210,14 @@ data:
         Time_Key time
         Time_Format %d/%b/%Y:%H:%M:%S %z
 
+    {{ if .Values.extraParsers -}}
+    # Extra parsers
+    {{- range .Values.extraParsers }}
+    [PARSER]
+        {{- . | nindent 8 }}
+    {{- end }}
+    {{- end }}
+
```

Nits: making the whitespacing more consistent
```suggestion
    {{- if .Values.extraParsers -}}
    # Extra parsers
    {{- range .Values.extraParsers }}
    [PARSER]
        {{- . | nindent 8 }}

    {{- end }}
    {{- end }}
```